fat/fat2/fat2.go

```diff
@@ -0,0 +1,5 @@
+package fat2
+
+import "github.com/Factom-Asset-Tokens/fatd/fat"
+
+const Type = fat.TypeFAT2
```

Comment:
In my packages I just moved this to the transaction.go file. got rid of fat1.go and fat0.go files